Project.toml

```diff
@@ -12,13 +12,7 @@ OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
 
 [compat]
-Cassette = "0.2.2"
-CodeTracking = "0.5.0"
-OrderedCollections = "1.1"
-Revise = "2.1.3"
-TimerOutputs = "~0.5.0"
-julia = "1.1"
```

Comment:
we need these still, but might need different versions

Reply:
Fixed in a new commit below. In particular, I now require Julia@1.6 since it's when changes in the AST occur, and Revise@3.1.5 to fix [this issue](https://github.com/timholy/Revise.jl/issues/569). I also updated Cassette and CodeTracking to accommodate new major versions.

Reply:
(I tested the package with both the minimal versions in the new project file and the newest versions on Julia 1.6.3)